packages/core/src/Stores/client-store.js

```diff
@@ -621,7 +621,9 @@ export default class ClientStore extends BaseStore {
 
     @computed
     get is_options_blocked() {
-        return isOptionsBlocked(this.residence);
+        const blockedCountry = isOptionsBlocked(this.residence);
+        if (blockedCountry === true) return blockedCountry;
+        return false;
```

Comment:
The `isOptionsBlocked` function always returns a boolean value. So should we check ` if (blockedCountry === true)`?
```isOptionsBlocked = country => blocked_options_countries.includes(country);```